### PR TITLE
Update to restore options sentence

### LINKED ISSUE #11865: Typo in Restore docs - kv_options_list
Liv Lobo (livlobo) commented:

Docs here: https://www.cockroachlabs.com/docs/v21.1/restore.html#options

Current text:
kv_option_list | Control your backup's behavior with these options.

Proposed text:
restore_options_list | Control the behavior of your restore with a comma-separated list of these options. (link to options list table)

### PR DESCRIPTION
**Pull request by kathancox**
Closes #11865 

Remnant left from maybe the synopsis diagram, but update `kv_options` to `restore_options_list` as per the diagram. Also updated the sentence that introduces the options to avoid any confusion.

### CODE REVIEW

**Review by livlobo — APPROVED** [review]
Thanks for fixing this!

File: v20.2/restore.md [hunk 1]
```diff
@@ -46,11 +46,11 @@ You can restore:
  `full_backup_location` | The URL where the full backup is stored. <br/><br/>For information about this URL structure, see [Backup File URLs](#backup-file-urls).
  `incremental_backup_location` | The URL where an incremental backup is stored.  <br/><br/>Lists of incremental backups must be sorted from oldest to newest. The newest incremental backup's timestamp must be within the table's garbage collection period.<br/><br/>For information about this URL structure, see [Backup File URLs](#backup-file-urls). <br/><br/>For more information about garbage collection, see [Configure Replication Zones](configure-replication-zones.html#replication-zone-variables).
  `AS OF SYSTEM TIME timestamp` | Restore data as it existed as of [`timestamp`](as-of-system-time.html). You can restore point-in-time data only if you had taken full or incremental backup [with revision history](take-backups-with-revision-history-and-restore-from-a-point-in-time.html).
- `kv_option_list` | Control your backup's behavior with [these options](#options)
+ `restore_options_list` | Control your backup's behavior with [these options](#options)
```
Comment (rmloveland) [inline v20.2/restore.md:49]: Missing period?

**Review by rmloveland — APPROVED** [review]
LGTM, maybe saw a missing period?

File: v20.2/restore.md (hunk 1 above)
Comment (kathancox) [reply v20.2/restore.md:49]: Ha yessss! Thank you

**Pull request merged**